docs/content/policy-reference.md

```diff
@@ -161,6 +161,17 @@ The following table shows examples of how ``glob.match`` works:
 | ``output := glob.match("{cat,bat,[fr]at}", [], "rat")`` | ``true`` | A glob with pattern-alternatives matchers. |
 | ``output := glob.match("{cat,bat,[fr]at}", [], "at")`` | ``false`` | A glob with pattern-alternatives matchers. |
 
+### Bitwise
+
+| Built-in | Description |
+| --- | --- |
+| <span class="opa-keep-it-together">``z := bits.or(x, y)``</span>   | ``z`` is the bitwise or of ``x`` and ``y`` |
```

Comment:
It might be good to be explicit and mention that `x` and `y` must be integers

Reply:
Do the updates look okay here?

Reply:
Yep, perfect!